src/solver/conv_hip_implicit_gemm_wrw_v4r4_xdlops.cpp

```diff
@@ -92,15 +92,15 @@ bool PerformanceImplicitGemmWrwV4R4Xdlops::SetNextValue()
             break;
         if(!NextTwoPower<1, 8>(GemmKPack))
             break;
-        if(!NextTwoPower<32, 128>(GemmNPerWave))
+        if(!NextTwoPower<4, 128>(GemmNPerWave))
```

Comment:
The actual range of GemmNPerWave is 16 to 128. But it is fine, the xdlopsgemm validation will check it.

Reply:
What about GemmMPerWave?

Reply:
Maybe we can enlarge Xdlop support to support GemmNPerWave=4 in the future? I will add a comment in the code

Reply:
I added some comment